backend/app/interior/application/interior_service.py

```
@@ -0,0 +1,216 @@
+import os
+import uuid
+import asyncio
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)